src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -203,7 +203,8 @@ def default_api_version(self):
         # Full MultiAPI support is not done in AppService, the line below is merely
         # to have commands show up in the hybrid profile which happens to have the latest
         # API versions
-        ResourceType.MGMT_APPSERVICE: '2018-02-01'
+        ResourceType.MGMT_APPSERVICE: '2018-02-01',
+        ResourceType.MGMT_EVENTHUB: '2017-04-01'
```

Comment:
I hope adding the api version here does NOT mandate , adding the api version to the latest profile as well.  pls confirm

Reply:
Latest profile already has the same definition. Please see https://github.com/Azure/azure-cli/blob/dev/src/azure-cli-core/azure/cli/core/profiles/_shared.py#L151